app/src/main/java/org/ole/planet/myplanet/ui/MyLife/LifeFragment.java

```diff
@@ -0,0 +1,110 @@
+package org.ole.planet.myplanet.ui.MyLife;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.ole.planet.myplanet.R;
+import org.ole.planet.myplanet.base.BaseRecyclerFragment;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link LifeFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link LifeFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class LifeFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+
+    // TODO: Rename and change types of parameters
+    private String mParam1;
+    private String mParam2;
+
+    private OnFragmentInteractionListener mListener;
+
+    public LifeFragment() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @param param1 Parameter 1.
+     * @param param2 Parameter 2.
+     * @return A new instance of fragment LifeFragment.
+     */
+    // TODO: Rename and change types and number of parameters
+    public static LifeFragment newInstance(String param1, String param2) {
```

Comment:
@jeevast  Don't include factory methods and interface callback. Just create blank fragment. We don't need those for now.